client/src/components/ui/radio-group.tsx

```diff
@@ -23,7 +23,7 @@ const RadioGroupItem = React.forwardRef<
     <RadioGroupPrimitive.Item
       ref={ref}
       className={cn(
-        'aspect-square h-4 w-4 rounded-full border border-primary text-primary ring-offset-background focus:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
+        'aspect-square h-4 w-4 rounded-full border border-primary text-primary ring-offset-background focus:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:border-gray-300 disabled:text-gray-300',
```

Comment:
I think the style is different here. Its supposed to be blue background with a white dot and a stroke line of 2
<img width="124" alt="image" src="https://github.com/Orcasa-Platform/orcasa/assets/9701591/afb1e219-3ff7-41cb-bb5d-7a7cef85e3b6">